.env.sample

```diff
@@ -0,0 +1,4 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_NAME=POSTGRES_NAME

```

Comment:
Due to documentation variable name must be POSTGRES_DB
For now your postgres container does not work